### ISSUE TITLE
Loop the map animation

### ISSUE THREAD

**Issue by nerik** [Vizzuality/GlobalFishingWatch#690]
https://airtable.com/tbliFUBZhHrWTVy5I/viwjB6S36hfjUz7EU/recNCb4tnUqHzG1Bd

**Comment by nerik** [comment]
BC issue: https://basecamp.com/1756858/projects/12049853/todos/309841995

**Comment by enriquetuya** [comment]
@nerik agreed, we should include the option in the next iteration time-bar. I've added the comment on the design scope document to take this into account when the time comes. I'll keep the ticket open for now. Unless you think we should close it.
